features/product/managing_product_variants/generating_product_variant_validation.feature

```diff
@@ -15,13 +15,13 @@ Feature: Generating product variant generation
         Given I want to generate new variants for this product
         When I specify that the 1st variant is identified by "WYBOROWA_ORANGE" code
         And I try to generate it
-        Then I should be notified that price is required for the 1st variant
+        Then I should be notified that prices in all channels must be defined for the 1st variant
```

Comment:
Prices should be required only for channels in which the product is available. What is more when we enable product in a specific channel we should validate that all it's variants have specified prices for this channel.